spynl/tests/test_mail.py

```diff
@@ -41,9 +41,16 @@ def config2(settings):
 @pytest.fixture
 def template(tmpdir):
     """Create a temporary template file for tests to use."""
-    file_ = tmpdir.mkdir('template').join(uuid4().hex + '.jinja2')
-    yield file_.strpath
-    os.remove(file_.strpath)
+    name = uuid4().hex
+    folder = tmpdir.mkdir('template')
+    file_ = folder.join(name + '.jinja2')
+    subject_ = folder.join(name + '.subject.jinja2')
+    yield (file_.strpath, subject_.strpath)
```

Comment:
why add .jinja2 if you're replacing it every time?

and why even replace it?

Reply:
Because the function takes the name without jinja2, but the inside the function it looks for the file with the .jinja2 extension. So I need a file that ends with .jinja2, but the argument is without .jinja2. I will take a look at it and see if I can make it more logical.